requirements.txt

```diff
@@ -35,6 +35,7 @@ oss2==2.18.5
 passlib<2.0.0
 pycryptodome
 pydash<8.0.0
+pyjwt
```

Comment:
interesting. did we miss to add this dependency earlier? if so: how come this was not an issue before? 🤔